pkgs/top-level/all-packages.nix

```
@@ -24272,12 +24272,7 @@ in
 
   rankwidth = callPackage ../development/libraries/science/math/rankwidth { };
 
-  fenics = callPackage ../development/libraries/science/math/fenics {
-    inherit (python3Packages) numpy ply pytest python six sympy;
-    pythonPackages = python3Packages;
-    pythonBindings = true;
-    docs = true;
-  };
+  fenics = python3Packages.fenics;
```

prevent python packages from polluting user env's
```suggestion
  fenics = with python3Packages; toPythonApplication fenics;
```

---

I was mainly doing this because there was already this top-level attribute. There isn't really any application that is provided. The main thing about fenics is the python library. I guess we remove that then?

---

if it's just the python module, you will want to ensure that the interpreter versions match, so it doesn't make sense to have a top-level attribute if it's not going to expose some command